Dockerfile

```diff
@@ -31,22 +30,28 @@ FROM elixir-builder as app-builder
 
 ENV LANG="C.UTF-8" MIX_ENV=prod
 
+RUN apk add --no-cache --update curl
+
 WORKDIR /root
 
+RUN curl https://truststore.pki.rds.amazonaws.com/global/global-bundle.pem \
+    -o aws-cert-bundle.pem
+
 # Add frontend assets compiled in node container, required by phx.digest
 COPY --from=assets-builder /root/priv/static ./priv/static
 
 RUN mix do compile --force, phx.digest, release
 
-FROM alpine:3.13.6
+FROM alpine:3.15.0
 
-RUN apk add --update libssl1.1 ncurses-libs bash curl dumb-init \
-  && rm -rf /var/cache/apk
+RUN apk add --update libssl1.1 libstdc++ libgcc \
```

Comment:
Suggestion (non-blocking): you can use --no-cache instead of update with a cache delete